ext/standaloneusers/css/standalone.css

```diff
@@ -1,5 +1,5 @@
 .standalone-page-padding {
-  padding: 1px 3vw 1rem;
+  padding: 1px 1vw 1rem;
```

Comment:
I'd suggest (if @artfulrobot agrees) just changing it to:

```
padding: 1px max(10px, 1vw) 1rem;
```

Reply:
(using max rather than clamp as the upper limit seems less important – `max` uses the larger of the two values).

Reply:
Second that. And if it looks too big on a 27 inch screen, we could go back to Clamp. ie:

```
padding: clamp(10px, 1.4vw, 26px);
```

Reply:
Thank you @vingle @artfulrobot - I updated the PR to use `max()`. It worked well on my 27in screen, smaller screen, and mobile device.